src/Control/Monad/Log.hs

```diff
@@ -43,7 +43,7 @@ module Control.Monad.Log
          logDebug, logInfo, logNotice, logWarning, logError, logCritical, logAlert, logEmergency,
 
          -- * Message transformers
-         PP.renderPretty,
+         PP.layoutPretty,
```

Comment:
I wonder if we should export an alias of `layoutPretty` as `renderPretty` and mark it as deprecated.

Reply:
Ah yea, that sounds reasonable to me.